### DISCUSSION TITLE
Seeing the phu: icons in the drop down with mdi: icons

### DISCUSSION THREAD

**Discussion by wolfspirituk** [elax46/custom-brand-icons#565] [Q&A] [answered]
I can use the "phu:eggs" example and I get the eggs in the UI.
Is there a way to get previews in the icon dropdown selectors all around the HA UI?
I find the icon prompter really nice to save remembering or looking up the codes.

**Comment by rchiileea** [comment, marked as answer]
https://github.com/elax46/custom-brand-icons/issues/560
